src/app/components/ProposalStatusIcon/index.tsx

```diff
@@ -36,21 +36,35 @@ export function ProposalStatusIcon(props: ProposalStatusProps) {
   const platformDayQuery = usePlatformDayQuery();
   const platformDay = platformDayQuery.data;
 
-  let startTimeStr = platformDay
-    ? dayjs()
+  const startTimeStr = () => {
+    if (platformDay && startDay >= platformDay) {
+      const startTime = dayjs()
         .utc()
         .add(startDay - platformDay, 'day')
-        .hour(17)
-        .fromNow(true)
-    : '';
+        .hour(17);
+      const timeLeft = startTime.diff(dayjs().utc(), 'hours');
+      if (timeLeft < 0) return '';
+      return startTime.fromNow(true);
+    }
+    return '';
+  };
 
-  let endTimeStr = platformDay
-    ? dayjs()
+  const endTimeStr = () => {
```

Comment:
Define the time formatting functions outside of the ProposalStautsIcon component scope.